.travis.yml

```diff
@@ -1,37 +1,54 @@
 ---
 language: python
 python:
-    - '3.6' # EOL 2021-12-23
-    - '3.7' # EOL 2023-06-27
-    - '3.8' # EOL 2024-10
-    - 3.9-dev # RC1 2020-09-10
+    - '3.6'  # EOL 2021-12-23
+    - '3.7'  # EOL 2023-06-27
+    - '3.8'  # EOL 2024-10
+    - 3.9-dev   # EOL 2025-10
```

Comment:
Doesn't this work with just '3.9' ? And adding 3.10-dev?

Reply:
Nope, not yet.  If you want to play with it feel free.  I'll poke at it next time I need to update this as well.